docs/content.rst

```diff
@@ -575,7 +575,7 @@ Adding Automatic Naming for Content
 
 On some sites you don't want to set the name for every piece of content you
 create. Substance D provides support for this with a special kind of folder.
-You can configure your site to use the autonaming folder like by overrding
+You can configure your site to use the autonaming folder by overriding
 the standard folder:
 
```

Comment:
Also it would be good to reference the post to the mail list: https://groups.google.com/d/msg/substanced-users/GW5uj4AT_J8/EGjLSjP4CAAJ and provide context of what is being overridden and edited, specifically a project generated from the substanced scaffold.

```rst
For example, in a project generated from the substanced scaffold,
simply add this at the top of ``resources.py``:
```